packages/teams-js/src/internal/communication.ts

```diff
@@ -180,12 +183,20 @@ export function sendMessageToParent(actionName: string, argsOrCallback?: any[] |
   }
 }
 
+const sendMessageToParentHelperLogger = communicationLogger.extend('sendMessageToParentHelper');
+
 /**@internal */
 function sendMessageToParentHelper(actionName: string, args: any[]): MessageRequest {
+  const logger = sendMessageToParentHelperLogger;
+
   const targetWindow = Communication.parentWindow;
   const request = createMessageRequest(actionName, args);
+
+  logger('Message %i information: %o', request.id, { actionName, args });
```

Comment:
How do we ensure the args do not contain EUII data?

Reply:
This is only ever logged to brower's developer console. I'm not sure having EUII here is really a concern. It might be, if we ask users to share these with us when reporting issues (e.g. as part of issues here on GitHub), but other than that it should be ok.

Reply:
Sounds good. I assume this log will be retrievable and collectable as part of the client logs?